ibft/event_loop.go

```diff
@@ -100,7 +100,7 @@ loop:
 					i.Logger.Info("found f+1 change round quorum, bumped round", zap.Uint64("new round", i.State.Round))
 				} else {
 					// if not found, wait 1 second and then finish to try again
-					time.Sleep(time.Second * 1)
+					time.Sleep(time.Millisecond * 100)
```

Comment:
`adding round change message to event queue` message is filling up the log, if we lower the sleep time we should consider removing the debug message

https://github.com/bloxapp/ssv/pull/251/files#diff-4a2cb99dd4d357389d2643805d526430052f572dc25f7c9d37557eac8b5b4b61R92